Lib/difflib.py

```diff
@@ -391,6 +393,8 @@ def find_longest_match(self, alo, ahi, blo, bhi):
         # the unique 'b's and then matching the first two 'a's.
 
         a, b, b2j, isbjunk = self.a, self.b, self.b2j, self.bjunk.__contains__
+        ahi = len(a) if ahi is None else ahi
+        bhi = len(b) if bhi is None else bhi
```

Comment:
```suggestion
        if ahi is None:
            ahi = len(a)
        if bhi is None:
            bhi = len(b)
```
It's a small point, but this is easier to follow the suggested way - which is the way similar defaults are resolved throughout the code base.  The "extra" lines don't matter at all.  What does matter is that it's visually obvious at once that "`is None`" is the _only_ thing going on.